### PR TITLE
Add css responsive table

### PR DESCRIPTION
**Pull request by bjgordon**
Table 1 in the Annual Report would suit a css responsive table approach so it's still readable on mobile. I've used the approach from https://css-tricks.com/responsive-data-tables/

### CODE REVIEW

**Comment by bjgordon** [comment]
Screenshot at mobile here: 
image

You can preview it in circle here: https://1347-71211972-gh.circle-artifacts.com/0/home/ubuntu/dta-website/_site/who-we-are/corporate/annual-report/2015-16/2-program-performance/index.html#results

File: _assets/scss/partials/_tables.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    td:before {
+      //Now like a table header
+      position: absolute;
+      //Top/left values mimic padding
+      top: $tiny-spacing;
+      left: $tiny-spacing;
+      width: 25%;
+      padding-right: 10px;
+      white-space: nowrap;
+      content: attr(data-label);
```
Comment (klepas) [inline _assets/scss/partials/_tables.scss]: Mhh… this is gonna suck for anyone wanting to select this for copy-pasting…

File: _assets/scss/partials/_tables.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    td:before {
+      //Now like a table header
+      position: absolute;
+      //Top/left values mimic padding
+      top: $tiny-spacing;
+      left: $tiny-spacing;
+      width: 25%;
+      padding-right: 10px;
```
Comment (klepas) [inline _assets/scss/partials/_tables.scss]: Why not use the spacing vars?

File: _assets/scss/partials/_tables.scss (hunk 1 above)
Comment (klepas) [inline _assets/scss/partials/_tables.scss]: Just tested this with VoiceOver: works fine it seems.

**Comment by TrebBrennan** [comment]
CSS all looks good to me assuming ie8+ and preview works as described.

Would recommend adding `vertical-align: top;` to your `td` for layout goodness.

File: _assets/scss/partials/_tables.scss (hunk 2 above)
Comment (bjgordon) [inline _assets/scss/partials/_tables.scss]: Fixed

**Comment by AndrewArch** [comment]
Agree with Trev's `vertical-align: top;`

IMHO this is a case where the 'data' can be reformatted visually (linearly) and still make sense. implementation looks fine. Just wondering if we can add insert 1/2/3/4 with the KPIs when the table linearises to indicate a visual flow through the rows - eg 'KPI-1' etc (but via CSS as the table structure is all there for a screen-reader users)

[Q - from an clarity/understanding perspective do 'portfolio performance measures' = 'KPIs'? (noting table cation vs table content terms)]

**Comment by bjgordon** [comment]
Have implemented all suggested changes from review comments.

**Pull request merged**